wagn/lib/wagn/config/routes.rb

```diff
@@ -0,0 +1,8 @@
+Wagn.application.routes.draw do
+
+  if !Rails.env.production? && Object.const_defined?( :JasmineRails )
+    mount Object.const_get(:JasmineRails).const_get(:Engine) => '/specs'
+  end
```

Comment:
This will fix the tests (I'm hoping).  It shouldn't be a problem for this to be here, since the file is now in the gem.